dttools/src/disk_alloc.c

```diff
@@ -0,0 +1,230 @@
+/*
+Copyright (C) 2015- The University of Notre Dame
+This software is distributed under the GNU General Public License.
+See the file COPYING for details.
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <sys/stat.h>
+#include <sys/mount.h>
+#include <string.h>
+#include <ctype.h>
+#include <errno.h>
+
+#include "disk_alloc.h"
+
+int disk_alloc_create(char *loc, char *space) {
```

Comment:
Pass in space simply as the number of bytes in an int64_t, this will make it easier to use the library.
In the test program, use `string_metric_parse` to convert from a string to an int64_t.